check_coverage.sh

```diff
@@ -20,6 +20,8 @@ RED_BG=$(tput setab 1)
 GREEN_BG=$(tput setab 2)
 NC=$(tput sgr0) # No Color
 
+make test || exit 1
```

Comment:
It is a nitpick, so you can ignore it if you want

This will make that `make before_commit` runs the tests twice (`before_commit` depends on `test` and will execute this script too).

I think it is better to just check if coverage.out exists instead of running "make test" without any condition

Reply:
The purpose is not to be sure that coverage.out exists, but to be sure it's up to date(before this workaround, the script did not actually check coverage as the name states)

Reply:
It's ok for me.

The only thing is that I just realized that the Jenkins configuration first run "make test-postgres" and then perform the check_coverage, so the coverage is checked using a different test. With your change, Jenkins will run first "make test-postgres", and then check_coverage, that will run "make test", so the coverage.out file will be overwritten. Probably it is not a problem at all, just guessing

Reply:
Actually, I fixed because it seemed to be a problem :D